client/spago.dhall

```diff
@@ -6,25 +6,19 @@ You can edit this file as you like.
 , dependencies =
   [ "aff"
   , "affjax"
+  , "argonaut-codecs"
```

Comment:
I noticed that a number of dependencies in this file appear to be outdated (we don't use `exists`, `foreign-generic`, `foreign`, `free`, `quickcheck`, and so on anywhere in the code). I've removed them and everything continues to build just fine, but I can replace any of these that we want explicitly.

Reply:
I’m not aware of any reason to keep dependencies here if we aren’t using them. 👍